rules/game_process.tex

```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}, straight over its head.
```

Comment:
Suggest rewording as:

The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands over their head.  The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}.

Question: Approx. how long does the referee hold this pose? Are there any constraints on where the referee is positioned or oriented while giving this signal?

Reply:
I was thinking to state to hold the position for two seconds. Considering robots should already looks at the referee. 
I want to make the detection of this signal realistic with the evolution of the rule. I can see a proposal for in game signal to take five seconds, from whistles to end of gestures.